### PR TITLE
Emails

### PR DESCRIPTION
**Pull request by JoelForamitti**
This PR creates a frontend interface for the members emails API

- [x] API and Store for emails
- [x] Datatables
- [x] Details & Create
- [x] Button in members table 
- [x] HTML Form for templates (@nanu-c, the line is marked in a comment below)

### CODE REVIEW

File: src/components/extensions/members/MembersTable.vue [hunk 1]
```diff
@@ -90,27 +92,34 @@ function copyEmails() {
...
+        summary: "Failure",
+        detail: "Could not copy emails to clipboard.",
+        life: 5000,
+      });
+    }
   );
 }
 
+// Send emails
+async function sendEmails() {
```
Comment (JoelForamitti) [inline src/components/extensions/members/MembersTable.vue:117]: This function is still a todo

File: src/components/datatable/ObjectDetail.vue [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+                :filter="true"
+                placeholder="Select multiple choices"
+              />
+            </div>
+
+            <div v-else-if="field.input_type === 'checkbox'">
+              <InputSwitchPrime v-model="object_temp[name]" />
+            </div>
+
+            <div v-else-if="field.input_type === 'html'">
```
Comment (JoelForamitti) [inline src/components/datatable/ObjectDetail.vue:187]: This part ist still a TODO

File: src/App.vue [hunk 3]
```diff
@@ -37,13 +37,14 @@ const { user } = storeToRefs(userStore);
   font-size: 14px;
 }
 
-.p-datatable.p-datatable-gridlines .p-datatable-tbody > tr > td, .p-datatable.p-datatable-gridlines .p-datatable-thead > tr > th{
```
Comment (nanu-c) [inline src/App.vue:40]: This should be in the `ObjectTable.vue` and not here

**Review by nanu-c — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 3 above)
Comment (JoelForamitti) [reply src/App.vue:40]: Done

**Review by nanu-c — APPROVED** [review]

**Pull request merged**